crates/node/src/roles/base.rs

```diff
@@ -17,23 +17,23 @@ use restate_core::{
     worker_api::ProcessorsManagerHandle,
     ShutdownError, TaskCenter, TaskKind,
 };
-use restate_types::net::node::{GetNodeState, NodeStateResponse};
+use restate_types::net::node::{GetPartitionsProcessorsState, PartitionsProcessorsStateResponse};
 
 pub struct BaseRole {
     processor_manager_handle: Option<ProcessorsManagerHandle>,
-    incoming_node_state: MessageStream<GetNodeState>,
+    processors_state_request_stream: MessageStream<GetPartitionsProcessorsState>,
```

Comment:
Should this move to PPM?

Reply:
If so, do we still need BaseRole or should we remove it?

Reply:
It will be eventually deleted but right now it still handles GetPartitionsProcessorsState messages needed for CC operation